packages/electrode-bolt/config/babel/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
NOTE: babelrc _needs_ to be in root project and cannot be "pointed to" from archetype. @exogen has a better handle on what's up with this problem.